litgpt/finetune/adapter.py

```diff
@@ -211,7 +211,7 @@ def fit(
         f" {model.max_seq_length} and context length is {model.config.block_size}"
     )
 
-    validate(fabric, model, val_dataloader, tokenizer, dataclasses.replace(eval, max_iters=2), data)  # sanity check
+    val_loss = validate(fabric, model, val_dataloader, tokenizer, dataclasses.replace(eval, max_iters=2), data)  # sanity check
```

Comment:
Notice that this only does 2 iters of validation. Showing a value here could be very misleading.

I wonder if people get confused because we show `n/a`. Maybe reverting to only showing the val loss value when we have computed it once would help in avoid this confusion.

Reply:
> Notice that this only does 2 iters of validation. Showing a value here could be very misleading.

That's fair. Would you be ok to run `validate` with the `eval.max_iters` here as done later? I think knowing the original, initial validation loss is extremely helpful because often the most improvement is made during the first steps. And this way you know what the performance of the original model was.

Also, the user confusion was not necessarily the n/a but the fact that the validation loss was never calculated because the finetuning finished in fewer steps than `eval.max_iters`. Right now, triggering validation requires a user to know the number of steps in advance and then set the `eval.max_iters` to a number smaller than that.